tasks/package.yml

```diff
@@ -7,13 +7,15 @@
   when: ansible_os_family == 'Debian'
 
 - name: Install the GPG key (Debian-like)
-  apt_key: file=/tmp/DUO-GPG-PUBLIC-KEY id=C1A07A85 state=present
+  apt_key:
+    url: https://duo.com/DUO-GPG-PUBLIC-KEY.asc
+    id: 08C2A645DDF240B85844068D7A450864C1A07A85
   become_user: root
   when: ansible_os_family == 'Debian'
 
 - name: Install the repo (Debian-like)
-  apt_repository: >
-    repo='deb http://pkg.duosecurity.com/{{ ansible_distribution }} {{ ansible_distribution_release }} main'
+  apt_repository:
+    repo: 'deb [arch=amd64] https://pkg.duosecurity.com/{{ ansible_distribution }} {{ ansible_distribution_release }} main'
```

Comment:
If changing to https transport, should add the `apt-transport-https` package, since it's not necessarily already installed.

Reply:
a-t-h is only required on xenial, newer versions of apt include https support natively.

https://packages.ubuntu.com/xenial/apt-transport-https
https://packages.ubuntu.com/search?keywords=apt-transport-https

I could add the dependency as a stop gap until Xenial support disappears, I assume mid next year (?)

Reply:
TIL. In that case, it does not seem worthwhile to special case in this package (aka, don't change anything).